scripts/babel-relay-plugin/src/RelayQLAST.js

```diff
@@ -381,6 +381,32 @@ class RelayQLArgument {
             this.type.ofType()
           )
         );
+      case 'ObjectValue':
+        return _recurValue(value);
+    }
+    // unlike ListValue, ObjectValue return a
+    // plain javascript object(key-value)
+    function _recurValue(astValue){
+      switch (astValue.kind){
+        case 'IntValue':
+          return parseInt(astValue.value, 10);
+        case 'FloatValue':
+          return parseFloat(astValue.value);
+        case 'StringValue':
+        case 'BooleanValue':
+        case 'EnumValue':
+          return astValue.value;
+        case 'ListValue':
+          return astValue.values.map(function (v){
+            return _recurValue(v);
+          });
+        case 'ObjectValue':
+          var ob={};
+          astValue.fields.map(function (field) {
+            ob[field.name.value] = _recurValue(field.value);
+          });
+          return ob;
+      }
     }
```

Comment:
Let's move this to the bottom of the file to make it clear that it's a helper and doesn't rely on any scope here. also, how about naming it something like `getInputObjectValue(inputObject)` to make it more clear what it's doing?